substrate/primitives/weights/src/weight_meter.rs

```diff
@@ -165,80 +152,80 @@ mod tests {
 	fn weight_meter_remaining_works() {
 		let mut meter = WeightMeter::with_limit(Weight::from_parts(10, 20));
 
-		assert!(meter.check_accrue(Weight::from_parts(5, 0)));
+		assert_eq!(meter.try_consume(Weight::from_parts(5, 0)), Ok(()));
```

Comment:
use `assert_ok`

Reply:
yeah, I didn't want to add an unnecessary dependency, but I'm guessing it's fine to add it?

Reply:
As dev dependency yes, as normal dependency not (since its a primitive crate).  
But its fine either way.